policy/modules/system/udev.te

```diff
@@ -45,7 +45,7 @@ ifdef(`enable_mcs',`
 # udev Local policy
 #
 
-allow udev_t self:capability { chown dac_override dac_read_search fowner fsetid mknod net_admin net_raw setgid setuid sys_admin sys_nice sys_nice sys_ptrace sys_rawio sys_resource };
+allow udev_t self:capability { chown dac_override dac_read_search fowner fsetid mknod net_admin net_raw setgid setuid sys_admin sys_nice sys_nice sys_ptrace sys_rawio sys_resource sys_tty_config };
```

Comment:
Why are you adding perms to udev, when this is moving console-setup into its own domain?

Reply:
Thanks, that was a mistake.